dll/opengl/opengl32/icdload.c

```diff
@@ -43,7 +43,7 @@ static PVOID APIENTRY wglGetCurrentValue()
     return IntGetCurrentICDPrivate();
 }
 
-static DHGLRC wglGetDHGLRC(struct wgl_context* context)
+static DHGLRC APIENTRY wglGetDHGLRC(struct wgl_context* context)
```

Comment:
This thing is casted to 'PROC', where does that come from?

Reply:
IIRC this is an stdcall vs cdecl issue, hence this addition.

Reply:
PROC is declared as stdcall here: https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/psdk/windef.h;hb=d3e83d7d5e43dd9949592c406637b260339f32c5#l270

However it's unclear how exactly this is being called. DrvSetCallbackProcs appears not not actually exist, so it can't be verified what calling convention it uses.

Reply:
Found a random piece of documentation that says the type for this callback is:
`typedef DHGLRC (APIENTRY * PFN_GETDHGLRC)(HGLRC hrc);`
So this change should be good.